frontend/src/pages/AboutUs/AboutUsPage.tsx

```diff
@@ -85,15 +85,33 @@ export default function AboutUsPage() {
           and mental and behavioral health. In conjunction with key strategic
           partners, SHLI enhances leadership among diverse learners, conducts
           forward-thinking research on the drivers of health inequities, and
-          advances evidence-based policies; all in an effort to contribute to
+          advances evidence-based policies all in an effort to contribute to
           the achievement of health equity for all population groups.
         </p>
 
+        <h3 className='font-sansTitle font-medium'>
+          Native Land Acknowledgment
+        </h3>
+
+        <p className='max-w-md px-6 text-left'>
+          The Health Equity Tracker and Morehouse School of Medicine
+          respectfully acknowledge that our work is conducted on the ancestral
+          lands of the Mvskoke (Muscogee) people, who have stewarded this land
+          since time immemorial. We recognize the enduring relationship between
+          the Mvskoke people and these lands and honor their past, present, and
+          future contributions to this region.<br/><br/> In acknowledging this history, we
```

Comment:
Best practice to not use `<br />` tags in regular text like this; if we want a break just split it into two different `<p></p>` sections